### ISSUE TITLE
/log issues

### ISSUE THREAD

**Issue by Claycorp** [jaredlll08/JSON-Ables#7]
/log - should have an output saying the options or that it fails or both. Currently it throws this http://pastebin.com/V49mRxBt

/log materials and fluids - Creates an empty file and saves nothing to it.

**Comment by jaredlll08** [comment]
What did you actually type? like what command did you type?

**Comment by Claycorp** [comment]
Really -.-

`/log` throws an error  
`/log materials` does not save the data....  
`/log fluids` does not save the data...

**Comment by jaredlll08** [comment]
Ah ok, reason I asked was because the error was because there were no
params, Can I get the log if the others? (Game log, not crash log) please

**Comment by Claycorp** [comment]
Yes... the `/log` has no params and when used it throws the error. Remove it, add a response to it telling people to use one of the other thing or something.

Data dumps work if you build from source but you need to update curseforge with a new version as its way behind.
